scripts/pvkeys-download.sh

```diff
@@ -4,25 +4,59 @@
 
 set -eo pipefail
 
-# Get fixed set of PV keys (which needs to be updated when snark changes)
-if [ -z "$JSON_GCLOUD_CREDENTIALS" ]; then
-    echo "WARNING: JSON_GCLOUD_CREDENTIALS not set, static PV keys not used"
-    exit 0
-fi
+# When running in CI
+if [ "$CI" = true ] ; then
+    # Get fixed set of PV keys (which needs to be updated when snark changes)
+    if [ -z "$JSON_GCLOUD_CREDENTIALS" ]; then
+        echo "WARNING: JSON_GCLOUD_CREDENTIALS not set, static PV keys not used"
+        exit 0
+    fi
 
-# GC credentials
-echo $JSON_GCLOUD_CREDENTIALS > google_creds.json
-/usr/bin/gcloud auth activate-service-account --key-file=google_creds.json
+    # GC credentials
+    echo $JSON_GCLOUD_CREDENTIALS > google_creds.json
+    /usr/bin/gcloud auth activate-service-account --key-file=google_creds.json
+fi
 
 # Debug output
 #set -x
 
 # Get cached keys
 echo "------------------------------------------------------------"
 echo "Downloading keys"
-PINNED_KEY_COMMIT=temporary_hack
-TARBALL="keys-${PINNED_KEY_COMMIT}-${DUNE_PROFILE}.tar.bz2"
-/usr/bin/gsutil cp gs://proving-keys-stable/$TARBALL /tmp/.
+
+set +e
+
+# Derive branch being merged in to
+# Usually a fix/feature branch PR won't have PV keys
+PR_NUMBER=`basename ${CIRCLE_PULL_REQUEST:-NOPR}`
```

Comment:
This will fail master builds right? Because no explicit PR. Maybe it doesn’t matter though

Reply:
won't fail master builds.  will just fall back to old keys.
see the list it tries.

is master still in use in git-flow?